File: Factors_RevOrder/Fact.go
/*
	Author : Pakshal Jain
	Date : 10/03/2022
	Program : To Print Factors of Entered Number in Reverse order
*/

//Requried Libraries
package main
import "fmt"

//Function To Print 
func RevFact(No int) {
	var i int = 0;

	fmt.Println("Factors of Entered Number in Reverse order is : ");
	for i = No - 1;i > 0;i-- {
		if No % i == 0 {
			fmt.Printf("%d\t",i);
		}
	}
}

//Main Function
func main() {
	var no int = 0;

	fmt.Println("Jay Ganesh...");

	fmt.Println("Enter One Number..");
	fmt.Scanf("%d",&no);

	//Call To RevFact Function
	RevFact(no);
}

File: Factors_RevOrder/Fact.py
"""
    Author : Pakshal Jain
    Date : 10/03/2022
    Program : To Print Factors of Entered Number in Reverse order
"""

#####################################################################################################################

def RevFact(No) :
    print("Factor of Entered Number in Reverse order is : ");
    for i in range(No-1,0,-1) :
        if(No % i == 0) :
            print(i,end = "\t");

#####################################################################################################

def main() :
    No = 0;

    print("Jay Ganesh...");
    print("Enter one Number....");
    No = int(input());

    //Call To RevFact Function
    RevFact(No);

###############################################################################################################

if __name__ == "__main__" :
    //Call To Main Function
    main();